source/onboard/run-bulk-loading-command.rst

```diff
@@ -6,8 +6,10 @@ Bulk load data
 
 Before running the bulk loading command, you must first create a `JSONL <https://jsonlines.org>`__ file that contains the data that you want to import in your Mattermost directory. The file can have any name, but in this example it's called ``data.jsonl``. The format of the file is described in the :ref:`data-format` section.
 
-After you create the JSONL file, you need to zip it by running the ``zip -r data.zip data.jsonl`` command, and upload the ZIP file to the database by running the `mmctl import upload </manage/mmctl-command-line-tool.html#mmctl-import-upload>`__ command. For example: ``mmctl import upload data.zip``.
+Next, zip it by running the ``zip -r data.zip data.jsonl`` command.
 
-Confirm that the file is uploaded and ready for use by running the `mmctl import list available </manage/mmctl-command-line-tool.html#mmctl-import-list-available>`__ command. 
+From Mattermost v9.5, the mmctl bulk import process command in `local mode </manage/mmctl-command-line-tool.html#local-mode>`__ supports processing an import file without uploading it to the server. Append ``--bypass-upload`` to the zip command: ``zip -r data.zip data.jsonl --bypass-upload`` to enable the Mattermost server to read from the file directly.
```

Comment:
This is incorrect. The `--bypass-upload` flag is added to the `mmctl` command, not the `zip` command. 

To clarify, from 9.5 onwards, in local mode, you just need to run 1 command - `mmctl import process --bypass-upload <file>.zip` to start your import. Before that you would need 2 commands, one to upload, and one to start processing.

Reply:
Thanks, @agnivade! I've incorporated corrections in my latest commit.